### PR TITLE
Mapping quantities (see #2533)

### PR DESCRIPTION
**Pull request by maxfb87**

### CODE REVIEW

File: src/blenderbim/blenderbim/bim/module/pset/qto_calculator.py [hunk 1]
```diff
@@ -1063,6 +1081,16 @@ def get_object_main_axis(self, o):
         else:
             return "x"
 
+    def is_opening_horizontal(self, o):
+        x = (Vector(o.bound_box[4]) - Vector(o.bound_box[0])).length
+        y = (Vector(o.bound_box[3]) - Vector(o.bound_box[0])).length
+        z = (Vector(o.bound_box[1]) - Vector(o.bound_box[0])).length
+
+        if z < x and z < y:
```
Comment (Amoki) [inline src/blenderbim/blenderbim/bim/module/pset/qto_calculator.py:1089]:
```python
return z < x and z < y
```
😇

Comment (maxfb87) [reply src/blenderbim/blenderbim/bim/module/pset/qto_calculator.py:1089]: What? :-)

Comment (aothms) [reply src/blenderbim/blenderbim/bim/module/pset/qto_calculator.py:1089]: @Amoki means `if a: return True; else: return False` is the same as `return a`.

Comment (maxfb87) [reply src/blenderbim/blenderbim/bim/module/pset/qto_calculator.py:1089]: Ahh now i see, thanks! I looked at it in a hurry and i didn't see the difference!

**Pull request merged**